### PR TITLE
article about sports event

### PR DESCRIPTION
**Pull request by jkingyens**
This is a draft sample for an article pertaining to a sporting event

### CODE REVIEW

File: src/50_Sample_AMPs/Sports_Article.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        "width": 922
+      }
+    }
+    </script>
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+  </head>
+  <body>
+  <h1>The Toronto Wildcats defeat the San Jose Stars 4-2</h1>
+  <amp-img src="/img/hockey.jpg" alt="Lorem Ipsum?" height="614" width="922"></amp-img>
```
Comment (sebastianbenz) [inline src/50_Sample_AMPs/Sports_Article.html]: Where does the image come from, should we add the `attribution` attribute?

**Comment by sebastianbenz** [comment]
Nice! Few comments:

* Can we make the sample look more like an actual sports article? Similar to news article?
* Enable preview and mark as draft:
```
<!---{
  "draft": true,
  "preview": true
  }--->
```
* The structured data testing tool shows some warnings for this sample, should we fix these?
```
warningdescription	
The description field is recommended. Please provide a value if available.
warningendDate	
The endDate field is recommended. Please provide a value if available.
warningimage	
The image field is recommended. Please provide a value if available.
warningoffers	
The offers field is recommended. Please provide a value if available.
warningperformer	
The performer field is recommended. Please provide a value if available.
warningurl	
The url field is recommended. Please provide a value if available.
```

**Comment by jmadler** [comment]
@cartland - please review the SD as you're more up-to-date than I am

File: src/50_Sample_AMPs/Sports_Article.html [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    <script type="application/ld+json">
+    {
+      "@context": "http://schema.org",
+      "@type": "NewsArticle",
+      "mainEntityOfPage": "https://www.ampbyexample.com/sample_amps/sports_article",
+      "headline": "The Toronto Wildcats defeat the San Jose Stars 4-2",
+      "url": "https://www.ampbyexample.com/sample_amps/sports_article",
+      "datePublished": "2016-04-01T12:00:00Z",
+      "dateModified": "2016-04-01T12:00:00Z",
+      "description": "The Toronto Wildcats defeat the San Jose Stars 4-2",
```
Comment (cartland) [inline src/50_Sample_AMPs/Sports_Article.html]: What do you think about making headline different than the description? "Rivals face off during biggest game of the season."

File: src/50_Sample_AMPs/Sports_Article.html [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        "@type": "Person",
+        "name": "Jeff Kingyens"
+      },
+      "publisher": {
+        "@type": "Organization",
+        "name": "Google",
+        "url": "https://google.com",
+        "logo": {
+          "@type": "ImageObject",
+          "url": "https://www.ampbyexample.com/img/logo2.png",
```
Comment (cartland) [inline src/50_Sample_AMPs/Sports_Article.html]: Will this be a valid URL?

File: src/50_Sample_AMPs/Sports_Article.html [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+            "streetAddress": "525 W Santa Clara St, San Jose, CA 95113, United States"
+          },
+          "name": "The SAP Center",
+          "description": "an indoor arena located in San Jose, California. Its primary tenant is the San Jose Sharks of the National Hockey League, for which the arena has earned the nickname \"The Shark Tank\"",
+          "telephone": "850-648-4200"
+        }
+      },
+      "image": {
+        "@type": "ImageObject",
+        "url": "https://www.ampbyexample.com/img/hockey.jpg",
```
Comment (cartland) [inline src/50_Sample_AMPs/Sports_Article.html]: Can we host an image with a valid URL?

File: src/50_Sample_AMPs/Sports_Article.html (hunk 3 above)
Comment (jkingyens) [inline src/50_Sample_AMPs/Sports_Article.html]: Yes once this goes to master it will be valid.

File: src/50_Sample_AMPs/Sports_Article.html (hunk 4 above)
Comment (jkingyens) [inline src/50_Sample_AMPs/Sports_Article.html]: this url will be valid once the sample is published

File: src/50_Sample_AMPs/Sports_Article.html (hunk 1 above)
Comment (jkingyens) [inline src/50_Sample_AMPs/Sports_Article.html]: the image comes from wikipedia i think. I can add attribution if thats the norm

**Comment by cartland** [comment]
Looks good, the rest of my comments are covered by @sebastianbenz.

**Comment by sebastianbenz** [comment]
Any updates on this PR?

**Comment by sebastianbenz** [comment]
@jkingyens friendly ping

**Comment by jkingyens** [comment]
@sebastianbenz I've made fixes suggested here. PTAL, thanks!

**Comment by sebastianbenz** [comment]
LGTM

* Is the metadata already used somewhere?
* Do you want to mention that the metadata differs from the official one?
* We now support variables in samples, you could replace all references to `https://www.ampbyexample.com` with `<%host%>`.

**Comment by kul3r4** [comment]
@jkingyens friendly ping

**Pull request closed without merging**